operator/pkg/apis/cassandra/v1beta1/cassandradatacenter_types.go

```diff
@@ -50,7 +50,7 @@ type ProgressState string
 // In the event that no image is found, an error is returned
 func getImageForServerVersion(server, version string) (string, error) {
 	const (
-		cassandra_3_11_6 = "datastax/cassandra-mgmtapi-3_11_6:v0.1.2"
+		cassandra_3_11_6 = "datastax/cassandra-mgmtapi-3_11_6:v0.1.4"
```

Comment:
if you merge master in, I think this will go away

Reply:
I rebased off master and updated to v0.1.5.